plugins/modules/rpm_ostree_pkg.py

```diff
@@ -137,6 +163,12 @@ def ensure(self):
             results['action'] = 'uninstall'
             cmd.append('uninstall')
 
+        # Add the options to the command line
+        if self.params['apply_live'] and self.state in ('present'):
```

Comment:
Why not:
```suggestion
        if self.params['apply_live'] and self.state == 'present':
```
?

Reply:
I took it from line 159 to stay uniform with the rest of the file but yes, it could be simplified

Reply:
Instead of repeating mistakes from existing code, please fix lines 159 and 162 as well.